docs/src/components/banner/AppFrameBanner.tsx

```diff
@@ -7,7 +7,7 @@ import { alpha } from '@mui/material/styles';
 export default function AppFrameBanner() {
   return FEATURE_TOGGLE.enable_docsnav_banner ? (
     <Link
-      href={ROUTES.careers} // Fix me!
```

Comment:
Hi @danilo-leal, I don't get what needs to be fixed here. Can you expand?

Reply:
I think that the comment is a reminder to not forget to update the link.